src/HotChocolate/Core/src/Types.Scalars/ScalarResources.resx

```diff
@@ -319,4 +319,14 @@
   <data name="IPv4Type_ValidationPattern" xml:space="preserve">
     <value>(^(?:(?:(?:0?0?[0-9]|0?[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])\.){3}(?:0?0?[0-9]|0?[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])(?:\/(?:[0-9]|[1-2][0-9]|3[0-2]))?)$)</value>
   </data>
+  <data name="RegularExpressionType_Description" xml:space="preserve">
+    <value>The Regular Expression scalar type represents a String Type following a regular expression pattern/format, as defined
+ here https://en.wikipedia.org/wiki/Regular_expression.</value>
+  </data>
+  <data name="RegularExpressionType_IsInvalid_ParseLiteral" xml:space="preserve">
+    <value>RegularExpressionType cannot parse the provided literal. The provided value does not match the regular expression pattern.</value>
```

Comment:
```suggestion
    <value>{0}Type cannot parse the provided literal. The provided value is of an invalid format.</value>
```

Reply:
This way it is hidden from the user that the scalar is based on a regex

Reply:
good point!